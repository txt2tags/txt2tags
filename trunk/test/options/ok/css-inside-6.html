<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML>
<HEAD>
<META NAME="generator" CONTENT="http://txt2tags.sf.net">

<!-- CSS include failed for missing.css -->

<!-- Included css -->
<STYLE TYPE="text/css">
p { color: blue; }
</STYLE>

</HEAD><BODY BGCOLOR="white" TEXT="black">
<CENTER>
</CENTER>

<P>
Text.
</P>

<!-- html code generated by txt2tags (http://txt2tags.sf.net) -->
<!-- cmdline: txt2tags -q -\-no-rc -t html -\-css-inside -\-style missing.css -\-style css css-inside-6.t2t -->
</BODY></HTML>
