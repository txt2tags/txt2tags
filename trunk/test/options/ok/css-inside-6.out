txt2tags: Error: CSS include failed for missing.css
